contexts/did-configuration-v0.0.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": [
+    "https://schema.org",
```

Comment:
It will when requested with the correct content headers I think... I pulled this from the did core spec: https://github.com/w3c/did-core/blob/master/contexts/did-v0.11.jsonld#L8

Reply:
This is not the same thing.

In JSON-LD, if the @context property may be
- a map for specifying context definition
- an IRI to specify the location of another context
- an array combining either of the above.

- In this file, the https://schema.org url is the first element of the @context array and is not a map, therefore it should resolve to a valid context. See https://w3c.github.io/json-ld-syntax/#the-context

- In the file you link, the "schema" value is used for [IRI expansion](https://w3c.github.io/json-ld-syntax/#iri-expansion-within-a-context).

Reply:
hmm, I can see that schema.org is indeed being resolved by the document loader in the linked data proof example, here is a snippet that I pulled from the document loader:

```
https://schema.org { contextUrl: null,
      documentUrl: 'https://schema.org/docs/jsonldcontext.jsonld',
      document:
```

That url does resolve to a context... however, we are not using any schema.org properties in this proof. so I should be able to remove it, then use a property and see the test fail, then add it back again.

Reply:
I was able to confirm that the context is indeed being used by adding the following to the domain link assertion:

```
 "@type": "WebSite",
        url: "https://identity.foundation.com/"
```

An error occurs when schema.org url is omited, and the url property above is included.

Reply:
I think we should remove schema.org since we are not using it here.

Reply:
Interesting, I wonder how the "https://schema.org/" url gets translated to the full path that contains the json-ld "https://schema.org/docs/jsonldcontext.jsonld"